src/site/content/documentation.adoc

```diff
@@ -1,4 +1,4 @@
-= Apache Shiro Documentation
+Apache Shiro Documentation
```

Comment:
Looking at the examples on https://asciidoc.org/, I think the equals sign must stay, no?

Reply:
I still don't get it, why did this change? the `=` sign denotes a top-level heading according to what I read in asciidoc. This was correct. This should not change, or am I missing something?

Reply:
Yes, I think @bmarwell is correct, the equal sign needs to stay.